.java-version

```diff
@@ -0,0 +1 @@
+11.0
```

Comment:
I'm not sure we want to version control this. I'm not sure what specific version "11.0" corresponds to, but it will complain if someone is using "openjdk-11.0.2".

Reply:
The issue that I was trying to solve is jumping between commits where the java version has changed. If there is no .java-version file, you'll need to manually set the jenv local version. If .java-version exists but you use the openjdk version, you'll still have to manually set the jenv local version to match the openjdk version, so it isn't a loss, right?

Reply:
Ah, I was running into an issue in the transform repo. They enforce a check for their specific java version. I just tested this locally and it works fine. I think we should do this to all the repos.